charmhelpers/contrib/storage/linux/lvm.py

```diff
@@ -60,7 +60,7 @@ def remove_lvm_physical_volume(block_device):
     '''
     p = Popen(['pvremove', '-ff', block_device],
               stdin=PIPE)
-    p.communicate(input='y\n')
+    p.communicate(input=b'y\n')
```

Comment:
Whilst I'm generally in agreement here, I'm a little worried this might be incompatible across versions of python3.  e.g. the original change was 8 years ago, so it's worked up till today.  I'm going to nack this from landing until we've verified that it works across py3.6 -> py3.9 (?py 3.10?).

Reply:
pvremove supports the `-y` or `--yes` flag from [xenial](https://manpages.ubuntu.com/manpages/xenial/en/man8/pvremove.8.html) through [impish](https://manpages.ubuntu.com/manpages/impish/en/man8/pvremove.8.html). That might be a better change for this.

Reply:
Yes, that does sound better, and it's similar to what `create_logical_volume` does via `lvcreate`. What do you think, @ajkavanagh ?

Reply:
I'm +1 on passing `--yes` and replacing the use of `Popen` with `subprocess.check_call`.